.gitlab-ci.yml

```diff
@@ -2,6 +2,11 @@ image: docker:git
 
 variables:
   DOCKER_REPOSITORY: mendersoftware/mender-convert
+  S3_BUCKET_NAME: mender-convert-images
+  MENDER_ARTIFACT_VERSION: master
+  MENDER_CLIENT_VERSION: master
+  RASPBIAN_URL: http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-09-30/2019-09-26-raspbian-buster-lite.zip
+  RASPBIAN_NAME: 2019-09-26-raspbian-buster-lite
```

Comment:
Have we thought about how to support more than one at some point? Between major Debian releases we might be in the situation where we have to. This is long in the future so just something to think about.

Reply:
Not deeply, but the method is extendable if we need it. New Debian versions are not the only reason to do this, but for example a Beaglebone image might also be introduced at some point. Right now, I prefer not to overcomplicate it as I think I would probably get it subtly wrong anyway.